requirements.txt

```diff
@@ -0,0 +1,4 @@
+-r services/utils/requirements.txt
+-r services/data/requirements.txt
```

Comment:
these two files got deleted in the dependency unpinning commit. What was the reason for completely removing module specific dependencies for the data and utils?

Reply:
removed